File: Hash Table/Hash Set/builtin_hash_set.py
hashset = set()

hashset.add(4)
hashset.add(2)
hashset.add(3)

hashset.remove(2)

print("2 not in hashset: {} ".format(2 not in hashset))

print("len: {}".format(len(hashset)))

for x in hashset:
    print(x)

File: Hash Table/Hash Set/BuiltinHashSet.java
import java.util.*;

public class BuiltinHashSet<K> {

    public void oper() {
        Set<K> hashSet = new HashSet<K>();

        hashSet.add((K)"test");
        hashSet.add((K)Integer.valueOf(1));
        hashSet.add((K)Integer.valueOf(4));
        hashSet.add((K)"hello");

        System.out.println(hashSet.size());

        System.out.println("Contains: " + 1 + hashSet.contains(Integer.valueOf(1)));
        System.out.println("Contains: " + 1 + hashSet.contains("Test"));

        System.out.println(hashSet.remove("test"));

        for(K item : hashSet) {
            System.out.println(item);
        }

    }

    public static void main(String[] args) {
        new BuiltinHashSet().oper();    
    }
}
